scripts/download-unicode-files.sh

```diff
@@ -30,7 +30,7 @@ for filename in $data_files ; do
 done
 
 for filename in $emoji_files ; do
-  curl -o "$UNIDIR/$filename" "$DOWNLOAD_URL_BASE/emoji/3.0/$filename"
+  curl -o "$UNIDIR/$filename" "$DOWNLOAD_URL_BASE/emoji/5.0/$filename"
```

Comment:
Yeah, you need curl -L to make curl follow it (I just submitted a PR to netrw, so that it will use -L by default)